### PR TITLE
Add kyverno 1.4 to matrix

### PR DESCRIPTION
**Pull request by vineethvanga18**
Signed-off-by: vineethvanga18 <reddy.8@iitj.ac.in>

Add version 1.4 to compatibility matrix in docs

### CODE REVIEW

**Comment by chipzoller** [comment]
Kyverno 1.4.0 isn't available yet. This should be held.

File: content/en/docs/Installation/_index.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ Currently, Kyverno runs as a single replica in the `Deployment` resource. Suppor
 |--------------------------------|----------------|----------------|
 | 1.2.1                          | 1.14           | 1.17           |
 | 1.3.0                          | 1.16           | 1.20*          |
+| 1.4.0                          | 1.16           | 1.20           |
```
Comment (chipzoller) [inline content/en/docs/Installation/_index.md:21]: 1.4.0 won't support 1.21? Seems like it should at this point.

**Review by chipzoller — COMMENTED** [review]
Should be ok to merge now there's a beta availability, but why is it Kubernetes 1.21 isn't listed as supported?

File: content/en/docs/Installation/_index.md (hunk 1 above)
Comment (vineethvanga18) [reply content/en/docs/Installation/_index.md:21]: We can add it if we are sure that it supports 1.21. Did you test this release with 1.21?

Comment (chipzoller) [reply content/en/docs/Installation/_index.md:21]: e2e tests are currently using KinD 0.10 which maps to Kubernetes 1.20. [This issue](https://github.com/kyverno/kyverno/issues/1973) created to bump to 1.21.

Comment (vineethvanga18) [reply content/en/docs/Installation/_index.md:21]: I think we are good to go with adding 1.21 @chipzoller

**Review by chipzoller — APPROVED** [review]

**Pull request merged**